diff --git a/node_modules/pusher/lib/pusher.js b/node_modules/pusher/lib/pusher.js
index 013a3ce..23d04a7 100644
--- a/node_modules/pusher/lib/pusher.js
+++ b/node_modules/pusher/lib/pusher.js
@@ -1,6 +1,4 @@
 const crypto = require("crypto")
-const url = require("url")
-
 const auth = require("./auth")
 const errors = require("./errors")
 const events = require("./events")
@@ -73,9 +71,10 @@ function Pusher(options) {
  * @returns {Pusher} instance configured for the URL and options
  */
 Pusher.forURL = function (pusherUrl, options) {
-  const apiUrl = url.parse(pusherUrl)
+  const apiUrl = new URL(pusherUrl)
   const apiPath = apiUrl.pathname.split("/")
-  const apiAuth = apiUrl.auth.split(":")
+  const apiAuth = apiUrl.username + ":" + apiUrl.password
+  const apiCredentials = apiAuth.split(":")
 
   return new Pusher(
     Object.assign({}, options || {}, {
@@ -83,8 +82,8 @@ Pusher.forURL = function (pusherUrl, options) {
       host: apiUrl.hostname,
       port: parseInt(apiUrl.port, 10) || undefined,
       appId: parseInt(apiPath[apiPath.length - 1], 10),
-      key: apiAuth[0],
-      secret: apiAuth[1],
+      key: apiCredentials[0],
+      secret: apiCredentials[1],
     })
   )
 }
